src/Mantasflowers.Persistence/Authentication/FirebaseContext.cs

```diff
@@ -0,0 +1,76 @@
+﻿using FirebaseAdmin.Auth;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Mantasflowers.Persistence.Authentication
```

Comment:
Would move this from `Persistence.Authentication` to `Services.AuthenticationService` or `Services.FirebaseService`